.docker/docker-compose.prod.yml

```diff
@@ -0,0 +1,9 @@
+version: '3.4'
+
+services:
+  app:
+    build:
+      target: prod
+      args:
+        image_tag: ${IMAGE_TAG:-master}
+    image: libero/article-page:${IMAGE_TAG:-master}
```

Comment:
I've never found a good default for `IMAGE_TAG`, but better than `latest` as it communicates how random this revision is